content/en/tracing/trace_pipeline/ingestion_controls.md

```diff
@@ -148,6 +148,22 @@ To specify a specific percentage of a service's traffic to be sent, add an envir
 4. Apply the appropriate configuration generated from these choices to the indicated service and redeploy the service. **Note**: The service name value is case sensitive. It should match the case of your service name.
 5. Confirm on the Ingestion Control Page that your new percentage has been applied by looking at the Traffic Breakdown column, which surfaces the sampling rate applied. The ingestion reason for the service is shown as `ingestion_reason:rule`.
 
+### Sampling precedence rules
+If sampling rules are set in multiple locations, the following precedence rule apply in order, where rules that appear first on the list can override lower precedence rules:
+
+1. Remotely configured sampling rules, either set through the [Agent](#managing-ingestion-for-all-services-at-the-agent-level) or [resource-based sampling][16]
```

Comment:
this should not be here.

Reply:
Happy to move it - where should it go?

Reply:
I mean remote configured sampling rules are only set through the "Resource-based Sampling" product. The agent managed ingestion does not change that.

Reply:
Gotcha - just updated